db/migrate/20210224230319_add_smal_image_url_to_ads.rb

```diff
@@ -0,0 +1,5 @@
+class AddSmalImageUrlToAds < ActiveRecord::Migration[5.0]
+  def change
+    add_column :ads, :small_image_url, :string, null: false
```

Comment:
what about `mobile_image_url` ? small not necessarily means it will be used in the mobile version of the site